router/router.go

```diff
@@ -114,4 +114,6 @@ func Docker(group *echo.Group, reg registry.Registry) {
 
 	// GET /v2/_catalog
 	group.Add(http.MethodGet, Catalog, reg.Catalog)
+	group.Add(http.MethodGet, "/search", reg.GetImageNamespace)
```

Comment:
lets put this in a const like we did with Catalog and the path can be `/v2/catalog/search`